cli/src/main/java/io/hyperfoil/cli/commands/WrkAbstract.java

```diff
@@ -117,18 +117,21 @@ public void mainMethod(String[] args, Class wrkClass) {
 
    //   @CommandDefinition(name = "wrk", description = "Runs a workload simluation against one endpoint using the same vm")
    public abstract class AbstractWrkCommand implements Command<HyperfoilCommandInvocation> {
-      @Option(shortName = 'c', description = "Total number of HTTP connections to keep open", required = true)
+      @Option(shortName = 'c', description = "Total number of HTTP connections to keep open", defaultValue = "10")
       int connections;
 
-      @Option(shortName = 'd', description = "Duration of the test, e.g. 2s, 2m, 2h", required = true)
+      @Option(shortName = 'd', description = "Duration of the test, e.g. 2s, 2m, 2h", defaultValue = "10s")
```

Comment:
Do both wrk and wrk2 have the same defaults?

Reply:
yes